.coveragerc

```diff
@@ -1,4 +1,8 @@
 [run]
-source = _pytest,testing
+source = pytest,_pytest,testing/
 parallel = 1
 branch = 1
+
+[paths]
+source = src/
+  .tox/*/lib/python*/site-packages/
```

Comment:
This on Windows would be `.tox\*e\Lib\site-packages`. Can we add another entry here and coverage will try one and then the other?

Reply:
Yes, added `.tox\*e\Lib\site-packages\`.

This gets used with `coverage report` - so you could test it after e.g. `tox -e py36-coverage -- testing/deprecated_test.py`.

Reply:
Works like a charm! 🤗